app/models/concerns/globalized.rb

```diff
@@ -18,6 +18,34 @@ def translates(*columns)
       super(*columns, fallbacks_for_empty_translations: true)
     end
 
+    def translates_rich_text(*columns)
+      translates(*columns)
+
+      columns.each do |col|
+        col = col.to_s
+        delegate col.to_sym, to: :translation
+        delegate "#{col}=".to_sym, to: :translation
+      end
+
+      after_save do
+        columns.each do |col|
+          translation.send(col).save if translation.send(col).changed?
+        end
+      end
+
+      self.const_get(:Translation).include(Module.new do
```

Comment:
Hier fände ich noch ein eigenes module im `Globalized` modul angebracht, vielleicht `Globalized::RichText`. Das wäre IMHO auch vom Naming her besser. Die Default scope Anpassung würde ich vielleicht noch mit einem Kommentar versehen, oder einfach weglassen und die includes in den 2 Controllern machen